src/wp-includes/block-template-utils.php

```diff
@@ -1014,6 +1023,19 @@ function _build_block_template_result_from_post( $post ) {
 		}
 	}
 
+	if ( 'wp_template' === $post->post_type ) {
+		$registered_template = WP_Block_Templates_Registry::get_instance()->get_by_slug( $post->slug );
```

Comment:
`WP_Post::$slug` is a dynamic class property. The `WP_Post` class doesn't have a `$slug` property declared.
Did you mean to use `$template->slug`?

Reply:
Another good catch, thanks, @anton-vlasenko! Fixed in 6a6394d7b073500a2817c82f0181c8ba69e39c94.